### PR TITLE
Add several environment variables

### PR DESCRIPTION
**Pull request by tmckernan**
Add several environment variables : 

- LOGIT_APPLICATION
- ENABLE_BASKET
- ENABLE_QUOTES
- ELASTICSEARCH_LIMIT
- CNET_FTP_ENDPOINT
- CNET_FTP_PORT
- CNET_FTP_USERNAME
- CNET_FTP_PASSWORD

### CODE REVIEW

File: terraform/modules/configs/deploy-all/main.tf [hunk 1]
```diff
@@ -159,6 +159,37 @@ data "aws_ssm_parameter" "browser_rollbar_access_token" {
 
 data "aws_ssm_parameter" "lb_public_alb_arn" {
   name = "${lower(var.environment)}-lb-public-alb-arn"
+
```
Comment (adrianmilne) [inline terraform/modules/configs/deploy-all/main.tf:163]: Getting a compile error as closing brace missing

**Comment by adrianmilne** [comment]
Can we maybe add these to the README too? Maybe with a comment explaining what they are for (we will have to start thinking about Operational Acceptance Testing/handover at some point soon - might make our lives easier down the line)

Think we may need a follow up PR at some point to convert some of the sensitive env vars to secrets (before we get to pen testing) - it was always the intention for the existing passwords - but never got time to do it - might add another ticket to review that - any thoughts @tommyb82 ?

File: terraform/modules/configs/deploy-all/main.tf (hunk 1 above)
Comment (tmckernan) [reply terraform/modules/configs/deploy-all/main.tf:163]: Issue should be resolved now

File: README.md [hunk 2]
```diff
@@ -20,26 +20,34 @@ Check the README file for details of how to create the database.
...
+  /bat/{env}-lograge-enabled - flag to enable log forrmating in spree
+  /bat/{env}-sendgrid-api-key - api key to sendgrid
+  /bat/{env}-mail-from - email address which email are sent from spree
+  /bat/{env}-aws_access_key_id - spree user access key id
+  /bat/{env}-aws_secret_access_key - spree user secret access key
+  /bat/{env}-sidekiq-concurrency - number of concurrent sidekiq workers
+  /bat/{env}-sidekiq-concurrency-searchkick - number of concurrent sidekiq workers for searchkick jobs
+  /bat/{env}-logit-node - url to logit.io elasticsearch cluster
+  /bat/{env}-browser-rollbar-access-token - client side rollbar token
+  /bat/sbx8-cnet-ftp-username - username to cnet ftp site
```
Comment (adrianmilne) [inline README.md]: Great update with the extra notes thanks. Minor point - these last few have `sbx8` explicit refs

Comment (tmckernan) [reply README.md]: No, sorry i have update the `sbx8` to `{env}`

**Review by adrianmilne — APPROVED** [review]
👍 Good stuff. As mentioned in the comment - I think we probably need a separate ticket to have a bit of a review of these parameters as we are getting to have quite a list now. I think some could maybe change to overridable defaults (e.g. like some of the flags and limits), and some should be made secrets. I'll put a ticket in JIRA to look at this as a follow up.

**Pull request merged**